eng/Version.Details.xml

```diff
@@ -112,22 +112,22 @@
       <Uri>https://github.com/dotnet/fsharp</Uri>
       <Sha>9904179c94c41d5d7f9008975d523d807b2a19c6</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NET.Test.Sdk" Version="16.9.0-preview-20210127-04" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.NET.Test.Sdk" Version="16.9.0-preview-20210129-05" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/microsoft/vstest</Uri>
-      <Sha>358ec6169542cf4c004d95cad1d3a45a8b8c4a6d</Sha>
+      <Sha>a523efd1ac8675b6f14f7dd3b96a6d8d47b75915</Sha>
     </Dependency>
     <Dependency Name="Microsoft.NET.ILLink.Tasks" Version="6.0.0-alpha.1.21076.1" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/mono/linker</Uri>
       <Sha>92b9b51bb8548320e40fc432efcb395156ad1445</Sha>
       <RepoName>linker</RepoName>
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.10.0-1.21078.8" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="3.10.0-1.21078.34" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/roslyn</Uri>
-      <Sha>b455d1c9ee26b29cf008c88ae0fe1d43721ff6e5</Sha>
+      <Sha>529f2e520b55c1c90fc530b92c101bce4506c33c</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Build" Version="16.10.0-preview-21076-16" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
@mmitche It looks like this downgrade is blocking the PR, any idea what happened here? I'm guessing a newer package was uploaded but with a lower version number?

Reply:
@dotnet/msbuild-admins should know the appropriate contributors to involve

Reply:
@Forgind you're on that list, correct❔ Who should we ping when something strange shows up in the `.NET 6 Dev`  Maestro++ channel from dotnet/msbuild❔

Hmm, this is a slightly odd case because dotnet/msbuild assets cross channel boundaries at the dotnet/sdk repo: Most dependencies use the `.NET 6 Dev` channel, Arcade uses `.NET Eng - Latest` (of course), but dotnet/msbuild and a few others use `VS 16.9`.  Oddly, `dotnet/roslyn` uses `VS Master` and Microsoft/vstest uses `VS 16.8`. Could that mix be part of the problem❔

Reply:
We re-versioned master for 16.10 early to get an arcade upgrade checked in sooner but didn't get our subscription set up correctly.  As such, 16.10 flowed into the sdk.  We fixed the subscriptions and this is the resolution to that. I don't know if there is a way to mark that failure as expected but I'll go ahead and use my admin privileges to merge this PR.